aepp/configs.py

```diff
@@ -367,3 +370,141 @@ def setSandbox(self,sandbox:str=None)->dict:
         self.header["x-sandbox-name"] = sandbox
         self.__configObject__["sandbox"] = sandbox
         return self.getConfigObject()
+
+    def setOauthV2setup(self,credentialId:str=None,orgDevId:str=None)->bool:
+        """
+        set the credential ID and the OrgIdDev as attributes of the instance.
+        * credentialId
+        * orgDevId
+        Returns True when success.
+        Argument:
+            credentialId : OPTIONAL : The credential id that can be found on your Project page.
+            orgDevId : OPTIONAL : the org Id but NOT the IMS string. It is defined on your project page.
+                Example : https://developer.adobe.com/console/projects/<orgId>/<projectId>/credentials/<credentialId>/details/oauthservertoserver
+        """
+        if self.connectionType != "oauthV2":
+            raise Exception("You are trying to set credential ID or orgDevId for auth that is not OauthV2. We do not support these auth type.")
+        if credentialId is None:
+            raise ValueError("credentialId is None")
+        if orgDevId is None:
+            raise ValueError("orgDevId is None")
+        self.credentialId = credentialId
+        self.orgDevId = orgDevId
+        return True
+    
+    def getSecrets(self,credentialId:str=None,orgDevId:str=None)->dict:
+        """
+        Access the different token available for your client ID.
+        If you did not use the setOauthV2setup, you can pass the required information as parameters.
+        Arguments:
+            credentialId : OPTIONAL : The credential id that can be found on your Project page.
+            orgDevId : OPTIONAL : the org Id but NOT the IMS string. It is defined on your project page.
+                Example : https://developer.adobe.com/console/projects/<orgId>/<projectId>/credentials/<credentialId>/details/oauthservertoserver
+        """
+        if self.connectionType != "oauthV2":
```

Comment:
should we move these checks into Utils instead

Reply:
I am not sure what part you are referring about. 
the configs part ? We can, we just need to change the reference everywhere. 
If it is in `configs.py` vs `utils.py` it does not matter. 
Let me know. 
I would believe that it would be a different PR to be honest.